sonar-project.properties

```diff
@@ -4,7 +4,7 @@ sonar.sources=.
 sonar.tests=.
 sonar.test.inclusions=**/*.spec.ts
 sonar.exclusions=**/*.js,jest.config.ts,globalSetup.ts,globalTeardown.ts,**/*.app.ts,**/seed-data/*.ts
-sonar.coverage.exclusions=**/board-management.uc.ts,**/*.module.ts,**/*.factory.ts
+sonar.coverage.exclusions=**/board-management.uc.ts,**/*.module.ts,**/*.factory.ts,**/deletion.console.ts

```

Comment:
why?

Reply:
It's like an app, also almost identical to https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/console/console.ts - can't really add a unit test for that.

Reply:
Well maybe it would be better to name it `deletion-console.app.ts` 🤔 . This way an additional exclusion wouldn't be needed as it would fall into the `**/*.app.ts` which is used for all the other apps.

Reply:
> Well maybe it would be better to name it `deletion-console.app.ts` 🤔 . This way an additional exclusion wouldn't be needed as it would fall into the `**/*.app.ts` which is used for all the other apps.

Done in https://github.com/hpi-schul-cloud/schulcloud-server/pull/4521/commits/4d1831398797583a91e57acce1ae14e6b40a1b36 .